src/plugins/.common/controllers/events/onGuildCreate.js

```diff
@@ -0,0 +1,6 @@
+const dbh = require("./../../structures/DataHandling/DatabaseHandler.js");
+
+exports.eventHandle = "guildCreate";
+exports.event = async (client, guild) => {
```

Comment:
Why pass it the client if it's not used?

Reply:
When this function is called all the way from here: https://github.com/dev-launchers-sandbox/project__discord-bot/blob/master/src/handler/Event.js#L10
It passed the client and the rest of the data, which in this case is the guild. We need to get the client first, or we won't be able to access the guild (param order).